config/initializers/content_security_policy.rb

```diff
@@ -25,3 +25,12 @@
 # For further information see the following documentation:
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only
 # Rails.application.config.content_security_policy_report_only = true
+
+Rails.application.config.content_security_policy do |policy|
+  policy.default_src :self
+  policy.style_src   :self, "'unsafe-inline'"
+  policy.img_src     :self, :data, :https
+  policy.script_src  :self, "'unsafe-inline'"
```

Comment:
It seems like there is just one inline script. Wouldn't it make sense to put that into a separate JavaScript file so we wouldn't need `unsafe-inline`?

Reply:
I was thinking the same, but Greenlight is using a lib to partially replace the document instead of reloading the page and this lib depends on inline scripts. Therefore I had to add the unsafe-inline. It's also documented in the commit with a ref to the related issue: https://github.com/bigbluebutton/greenlight/pull/1164/commits/fe1b514764cda184561957609a7727e77c93a609